lib/karo/db.rb

```diff
@@ -59,7 +59,15 @@ def load_local_db_config
       local_db_config = YAML.load(File.read('config/database.yml'))
 
       if local_db_config["development"].nil? || local_db_config["development"].empty?
-        raise Thor::Error, "Please make sure MySQL development configuration exists within this file? '#{local_db_config_file}'"
+        raise Thor::Error, "Please make sure development configuration exists within this file? '#{local_db_config_file}'"
+      end
+
+      if local_db_config["development"]["username"].nil?
```

Comment:
These additions seem unnecessarily restricting on how to load the database configuration. Shouldn't the adapter(s) already cover a variety of configuration options?

Reply:
@mattr  I added these because the `dropdb`, `createdb` and `pg_restore` commands that karo currently generates relies on these bits of data, but the error messages you get if they're not present isn't at all clear, and requires you to dig into the `karo` codebase to figure things out. 

This makes it very explicit without introducing any risk, in the spirit of incremental improvement. 

It'd be great down the track to have a sensible defaults for these (localhost for `host`, and perhaps the system username for `user`, but those changes would require another pull request and a lot of testing which we don't currently have time for.

Reply:
As discussed, added comment explaining the need for these new error messages so we can clean it up later, perhaps with sensible defaults for host and username.